pytorch_lightning/core/lightning.py

```diff
@@ -162,6 +166,30 @@ def automatic_optimization(self) -> bool:
         """
         return True
 
+    @property
+    def invalid_loss_strategy(self) -> Union[str, int]:
+        """
+        This property is used to decide behaviour on ``training_step`` result object
+        when using LightningDistributedDataParallel.
+
+        Can either be "skip_if_any" or "never_skip".
+
+        "skip_if_any":
+            If result is None on at least on, it will skip
+            ``backward`` and ``optimizer_step`` for all processes
+
+        "never_skip":
+            If result loss is NaN or Inf on 1 or more processes,
+            those processes will skip backward and gradient will be
+            synchronized before running ``optimizer.step``
+
+        """
+        return self._invalid_loss_strategy
+
+    @invalid_loss_strategy.setter
+    def invalid_loss_strategy(self, invalid_loss_strategy):
+        self._invalid_loss_strategy = invalid_loss_strategy
```

Comment:
I like using a setter. We can run the input validation here:

```suggestion
    def invalid_loss_strategy(self, invalid_loss_strategy):
        allowed = list(InvalidLossStrategy)
        if invalid_loss_strategy not in allowed:
            raise MisconfigurationException(
                f'LightningModule `invalid_loss_strategy` property should be within {allowed}. Found {invalid_loss_strategy}'
            )
```